src/native/eventpipe/ep-file.c

```diff
@@ -317,7 +317,7 @@ ep_file_alloc (
 	instance->stream_writer = stream_writer;
 	instance->format = format;
 
-	instance->event_block = ep_event_block_alloc (100 * 1024, format);
+	instance->event_block = ep_event_block_alloc (1024 * 1024, format);
```

Comment:
I'd recommend reverting this part of the change. I assume it is unneeded now and it may break readers/change performance characteristics in undesired ways.